src/main/resources/portal.properties.EXAMPLE

```diff
@@ -206,25 +206,13 @@ ucsc.build=hg19
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
 
-# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
-# Set any of these properties to enable the respective menu options in oncoprint:
+# OncoPrint driver mutation annotations
 # oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
 # oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
-
-# set this to false to disable the automatic selection of the custom driver/passenger mutations annotation (binary) in the oncoprint 
-# when custom annotation data is present.
-# oncoprint.custom_driver_annotation.default=false
-
-# set this to false to disable the automatic selection of the custom tiers in the oncoprint when custom annotation data is present. By default
-# this property is true.
-# oncoprint.custom_driver_tiers_annotation.default=false
-
-# OncoKB and Hotspots are automatically selected as annotation source. If you want to disable them, set the following property to false. If you
-# want them to be selected only if there are no custom annotation driver and passenger mutations, type "custom".
+# oncoprint.custom_driver_annotation.default=true
+# oncoprint.custom_driver_tiers_annotation.default=true
 # oncoprint.oncokb_hotspots.default=custom
+# oncoprint.hide_passenger.default=false
 
-# Set this to true if you want to check the "hide Putative Passenger Mutations" checkbox by default.
-# oncoprint.hide_passenger.default=true
-
-# Change this to custom file (e.g. file:/<path>) to have a custom set of genes in query page:
-# querypage.setsofgenes.location =
\ No newline at end of file
+# Custom gene sets
```

Comment:
why remove the example described in the text `file:/<path>`?

Reply:
I wanted to add more documentation because knowing the example was insufficient. For example, you still needed to know the format of the JSON file. 

But the complete description was too long for the portal.properties I thought. So I moved it everything to: https://github.com/thehyve/cbioportal/blob/f55ee15f68c8ba9d2d1a1c0210bfe79c1babd28e/docs/portal.properties-Reference.md#gene-sets-used-for-gene-querying

Reply:
I agree that the example should read:
```
# Change this to custom file (e.g. file:/<path>) to have a custom set of genes in query page:		
#querypage.setsofgenes.location=file:/<path>
```

Reply:
done.